README.md

```diff
@@ -77,8 +82,25 @@ You can pass the following options to `cmake ..`.
 * `-DUNDEFINED_BEHAVIOR_SANITIZER` Build with UndefinedBehaviorSanitizer
 * `-DALIGNED_MEMORY_MODEL` Build for aligned memory model only devices. Default is OFF.
 
-### Build
-To build the library run make in the build directory you executed CMake.
+#### To Include JNI
+
+JNI examples are NOT built by default.  If you wish to build JNI you MUST add -DBUILD_JNI=TRUE when running cmake:
+
+```
+cmake -DBUILD_JNI=TRUE
+```
+
+#### To Include Building GStreamer Sample Programs
+
+The GStreamer examples are NOT built by default.  If you wish to build them you MUST add -DBUILD_GSTREAMER_PLUGIN=true when running cmake:
```

Comment:
Thank you for the change! Just one small thing. Can you modify this from `The GStreamer examples` to `The GStreamer plugin and samples`? Also, for uniformity, can you replace this from `-DBUILD_GSTREAMER_PLUGIN=true` to `-DBUILD_GSTREAMER_PLUGIN=TRUE` (true in all caps). Thanks again!